test/integration/acceptance/get_roles_test.cpp

```diff
@@ -4,53 +4,46 @@
  */
 
 #include <gtest/gtest.h>
-#include "integration/acceptance/acceptance_fixture.hpp"
-#include "framework/integration_framework/integration_test_framework.hpp"
 #include "backend/protobuf/transaction.hpp"
+#include "framework/integration_framework/integration_test_framework.hpp"
 #include "framework/specified_visitor.hpp"
+#include "integration/acceptance/acceptance_fixture.hpp"
 #include "interfaces/permissions.hpp"
 
-
 using namespace integration_framework;
 using namespace shared_model;
 
-class GetRoles : public AcceptanceFixture {
-public:
-//    auto makeUserWithPerms(const interface::RolePermissionSet &perms = {
-//            interface::permissions::Role::kGetRoles}) {
-//        return AcceptanceFixture::makeUserWithPerms(perms);
-//    }
-
-};
+class GetRoles : public AcceptanceFixture {};
```

Comment:
There is no need in empty `GetRoles` class. Remove it. Use `AcceptanceFixture` below instead of `GetRoles`.